### PR TITLE
Cals 5569 Minimum Required Indication in UI for RFA 1C

### PR DESCRIPTION
**Pull request by vladimirPe**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5569

#### Brief feature description
- As a user, I want to know what fields are required in order to submit the RFA 01C form so that I can complete the application accurately the first time.

#### Does this PR block/effect other stories?
- No

#### Do any commands needs to be run in order to run the project with this PR?
- No

#### Code coverage percent diff as a result of this PR?
- No

### CODE REVIEW

File: test/javascript/rfa_forms/rfa01c_edit_view/desiredChildcard_tests.js [hunk 1]
```diff
@@ -132,6 +133,14 @@ describe('Verify RFA 01C child desired', function () {
       [{applicant_id: 20, relationship_to_applicant_freeform: 'test', relationship_to_applicant: {value: '', id: 0}}])
   })
 
+  it('verify required fields labels', () => {
+    let componentHtml = childCardComp.html()
+    expect(componentHtml).toContain('Is the child currently in your home?' + RfaCommon.requiredIndicator)
```
Comment (ratneshraval) [inline test/javascript/rfa_forms/rfa01c_edit_view/desiredChildcard_tests.js]: These kind of tests do not really check if correct item has correct label. It just check that somewhere in the HTML the text is found. It is better to find the individual item by id/class and check the text/label to make sure correct item has correct text.

Comment (vladimirPe) [reply test/javascript/rfa_forms/rfa01c_edit_view/desiredChildcard_tests.js]: fixed for: 
- First Name
- Last Name
- Date Of Birth

but for "Is the child currently in your home?" there is no id, class or for an attribute which can be used as a unique identifier. 
If you have any idea how to identify this label please share the information

Comment (ratneshraval) [reply test/javascript/rfa_forms/rfa01c_edit_view/desiredChildcard_tests.js]: That's good. At least we are isolating to only 1 component. Thanks for fixing the others.

**Comment by ratneshraval** [comment]
jenkins, retest this please

**Pull request merged**